run.bash

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+source setenv.bash
+pushd src/github.com/getlantern/flashlight
+go build
```

Comment:
Probably safer to say `go build && ./flashlight` so that it doesn't run flashlight when the build fails.